set.mm

```diff
@@ -303875,6 +303878,10 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
   althtmldef "^" as '&uarr;';
   latexdef "^" as "\uparrow"; /*
   latexdef "^" as "\widehat{\ }"; */
+htmldef "^r" as "<IMG SRC='uparrow.gif' WIDTH=7 HEIGHT=19 ALT='^' TITLE='^'>" +
+    "<IMG SRC='r.gif' WIDTH=7 HEIGHT=19 ALT='r' TITLE='r'>" ;
```

Comment:
I don't know what r.gif is, or whether it exists (or how to make it). So the width is probably wrong.